tests/tests_pytorch/trainer/test_trainer.py

```diff
@@ -714,7 +714,10 @@ def test_checkpoint_path_input_last(tmpdir, ckpt_path, save_last, fn):
 
         with ctxt:
             trainer_fn(ckpt_path=ckpt_path)
-        assert trainer.ckpt_path == final_path
+        if final_path is None:
+            assert trainer.ckpt_path == final_path
+        else:
+            assert mc._fs._strip_protocol(trainer.ckpt_path) == Path(final_path).as_posix()
```

Comment:
The test here isn't failing without the fix. It never gets a path with a protocol. We need to write a new test where we pretend to be loading from a path with a protocol. 

A unit test in `test_model_checkpoint.py` that calls `_find_last_checkpoints` should be enough to cover this case. Should I help with that?